public/pages/feed.html

```diff
@@ -24,174 +26,195 @@
-        </ul>
-      </div>
-      <div class="col-6 col-md">
-          <h5 class="title text-secondary">
-            Canais de Atendimento:
-          </h5>
-          <p>De segunda á sexta-feira: 08:00 ás 20:00</p>
-          <img class="img-fluid" src="/public/assets/images/contato.png"> +55 00 0000-0000 </a>
-          <p><img src="/public/assets/images/email.png"> recreio.org@gmail.com </a> </p>
-      </div>
-      <div class="col-6 col-md">
-          <h5  class="title text-secondary">
-            Follow us:
-          </h5>
-          <ul class="list-unstyled text-small">
-          <a href="https://github.com/Squad-61/" target="_blank"> <img src="/public/assets/images/github.png"> </a>
-          <a href="https://www.instagram.com/" target="_blank"> <img src="/public/assets/images/instagram.png"> </a>
-          <a href="https://pt-br.facebook.com//" target="_blank"> <img src="/public/assets/images/facebook.png"> </a>
-          <a href="https://br.linkedin.com/" target="_blank"> <img src="/public/assets/images/linkedIn.png" </a>
-          <a href="https://www.youtube.com/" target="_blank"> <img src="/public/assets/images/youtube.png"> </a>
-         </ul>
-      </div> 
-    <hr>
-    <section class ="d-flex justify-content-center align-items-center">
-      &copy: Squad 61.
-      <img width="60px + 15vw" src="/public/assets/images/logosquad.png">
-    </section>
-    </footer>
-    </div> 
-</footer> 
+    <div class="col">
+      <h5 class="title text-secondary">
+        Termos:
+      </h5>
+      <ul class="list-unstyled text-small">
+        <li><a class="text-muted" href="https://recodepro.org.br/" target="_blank">Recode Pro</a></li>
+        <li><a class="text-muted" href="#" target="_blank">Termos e Condições</a></li>
+        <li><a class="text-muted" href="#" target="_blank">Política de Privacidade</a></li>
+      </ul>
+    </div>
+    <div class="col">
+      <h5 class="title text-secondary">
+        Canais de Atendimento:
+      </h5>
+      <p>De segunda á sexta-feira: 08:00 ás 20:00</p>
+      <img class="img-fluid" src="../assets/images/contato.png"> +55 00 0000-0000 </a>
+      <p><img src="./public/assets/images/email.png"> recreio.org@gmail.com </a> </p>
+    </div>
+    <div class="col">
+      <h5 class="title text-secondary">
+        Follow us:
+      </h5>
+      <ul class="list-unstyled text-small">
+        <a href="https://github.com/Squad-61/" target="_blank"> <img src="/public/assets/images/github.png"> </a>
+        <a href="https://www.instagram.com/" target="_blank"> <img src="/public/assets/images/instagram.png"> </a>
+        <a href="https://pt-br.facebook.com//" target="_blank"> <img src="/public/assets/images/facebook.png"> </a>
+        <a href="https://br.linkedin.com/" target="_blank"> <img src="/public/assets/images/linkedIn.png"> </a>
+        <a href="https://www.youtube.com/" target="_blank"> <img src="/public/assets/images/youtube.png"> </a>

```

Comment:
Sugestão de correção:


```suggestion
      <p><img src="../assets/images/email.png"> recreio.org@gmail.com </a> </p>
    </div>
    <div class="col">
      <h5 class="title text-secondary">
        Follow us:
      </h5>
      <ul class="list-unstyled text-small">
        <a href="https://github.com/Squad-61/" target="_blank"> <img src="../assets/images/github.png"> </a>
        <a href="https://www.instagram.com/" target="_blank"> <img src="../assets/images/instagram.png"> </a>
        <a href="https://pt-br.facebook.com//" target="_blank"> <img src="../assets/images/facebook.png"> </a>
        <a href="https://br.linkedin.com/" target="_blank"> <img src="../assets/images/linkedIn.png"> </a>
        <a href="https://www.youtube.com/" target="_blank"> <img src="../assets/images/youtube.png"> </a>
```